### PR TITLE
Fixing grammatical errors in the CODE_OF_CONDUCT.md file. 

### LINKED ISSUE #1770: Updated grammatical errors in the CODE_OF_CONDUCT.md file Request
Code of Conduct file has 4 grammatical Errors.
I'm proposing to fix them.

### PR DESCRIPTION
**Pull request by msj2**
Fixing grammatical errors.

Congrats on taking the first step to contributing to the Sample Programs repository maintained by [The Renegade Coder][1]! For simplicity, please make sure that your pull request includes one and only one contribution.

## Complete the Applicable Sections Below

Find which section best describes your pull request and make sure you fill it out. To start, let us know which issue you've fixed.

- [x] I fixed #1770

### Code Snippets

- [ ] I named the pull request using `Added/Updated <Sample Program> in <Language>` format
- [X] I created/updated the language README
  - [ ] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])

### Documentation

- [X] I named the pull request using `Added/Updated <Sample Program> in <Language> Article` format
- [X] I followed the applicable article template
  - [ ] I added/updated a [language article][4] (i.e. The Python Programming Language)
  - [ ] I added/updated a [code snippet article][3] (i.e. Hello World in Perl)
  - [ ] I added/updated a [project article][5] (i.e. Fizz Buzz in Every Language)

### Testing

- [X] I named the pull request using `Added/Updated <Language>/<Project> Testing` format
- [ ] I followed the testinfo template, if applicable

## Notes

Feel free to describe what you added or updated.
Code of Conduct had 4 grammatical issues. These were updated.

[1]: https://therenegadecoder.com/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#plagiarism
[3]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/CODE_ARTICLE_TEMPLATE.md
[4]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/LANGUAGE_ARTICLE_TEMPLATE.md
[5]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/PROJECT_ARTICLE_TEMPLATE.md

### CODE REVIEW

**Comment by stuin** [comment]
I noticed some additional mixups on lines 25 and 39, and 201 has multiple possible awnsers, but otherwise it all looked good to me.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ Within each language folder, you'll find the following:
 -   A testinfo.yml
 
 Now, each program file maps to an ongoing project that you can find in the Docs directory.
-As for the README, it contains a list of the project files that
-link to existing articles in the documentation. In addition, the README contains
+As for as the README, it contains a list of the project files that
```
Comment (jrg94) [inline .github/CONTRIBUTING.md]:
```suggestion
As for the README, it contains a list of the project files that
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -186,32 +186,31 @@ At any rate, let's have some fun!
 ### Articles
 
 As a bonus to this repository, there are associated articles for every script.
-However, the articles do take awhile to write, so you can help by writing them
+However, the articles do take a while to write, so you can help by writing them
```
Comment (jrg94) [inline .github/CONTRIBUTING.md]: I tried looking into this one, and I have no idea which version is correct. haha

I'll take your word for it.

Source: https://www.merriam-webster.com/words-at-play/awhile-usage

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -186,32 +186,31 @@ At any rate, let's have some fun!
...
 as well.
 
 Previously, we had hosted all of the articles on [The Renegade Coder][34], but
 we've since moved away from that to support open-source editing. Now, you
 can add and update any article you want.
 
-Currently, there are three types of articles you can write: code, project, and
+Currently, there are three types of articles you can write: code, project and
```
Comment (jrg94) [inline .github/CONTRIBUTING.md]: I actually prefer the Oxford comma, but it doesn't really matter.

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -186,32 +186,31 @@ At any rate, let's have some fun!
...
 language. You can find out more about this type of articles below.
 
 #### Code
 
-To get started, you'll want to create a new markdown file (i.e. `python.md`)
+To get started, you'll want to create a new markdown file (eg: `python.md`)
```
Comment (jrg94) [inline .github/CONTRIBUTING.md]: This is interesting! I use i.e. a lot, so I had to look this one up: https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -18,11 +18,11 @@ for as many languages as possible.
 
 ## Repository Structure
 
-Before we get into the contribution rules, we should probably get an understanding for
+Before we get into the contribution rules, we should probably get an understanding of
 how this repository is structured.
 
 At the root of the repository, there are several housekeeping files that shouldn't matter
-much to the average user. The only item that matters is the archives and the docs folder.
+much to the average user. The only items that matters are the archives and the docs folder.
```
Comment (jrg94) [inline .github/CONTRIBUTING.md]:
```suggestion
much to the average user. The only items that matter are the archives and the docs folder.
```

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (msj2) [reply .github/CONTRIBUTING.md]: Accepting this.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (msj2) [reply .github/CONTRIBUTING.md]: articles do take a while to write is the proper one.

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (msj2) [reply .github/CONTRIBUTING.md]: i.e., is used to clarify
Here, we are giving an example of file, ...
Hence, eg is more apt.

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (msj2) [reply .github/CONTRIBUTING.md]: Accepting this, ..

**Pull request merged**